Update `@meiversion` for MEI basic

lpugin opened the pull request:
Because this will now be a version 5.0 release, the version attribute needed to be updated accordingly.

lpugin committed:
Update `@meiversion` for MEI basic

Because this will now be a version 5.0 release, the version attribute needed to be updated accordingly.

musicEnfanthen reviewed (commented):
Thanks @lpugin. I am fine with it, too. 

Just wondering if resp. want to make sure that the "Release Candidate" status is still valid/needed? Can someone clarify this question?

lpugin commented:
I was wondering too. Having `5.0.0-dev+basic` would be better in my opinion too. I'll change to that.

lpugin committed:
Update version to `-dev` instead of `+rc1`

We have no release candidate concept in MEI workflow, do `-dev` in more inline with what we do with the schema.
